pkg/controller/nodes/task/resourcemanager/config/config.go

```diff
@@ -34,6 +34,9 @@ type Config struct {
 
 // Specific configs for Redis resource manager
 type RedisConfig struct {
+	HostPaths   []string `json:"hostPaths" pflag:",Redis hosts locations."`
+	PrimaryName string   `json:"primaryName" pflag:",Redis primary name."`
```

Comment:
Can you add a comment for primary name?  what is a primary name?

Reply:
ohh should we just delete the other config attributes then? and use index 0 as the hostpath?

Reply:
Where can we find the primary name from the redis cluster?

Reply:
@kumare3  I thought for back compat we shouldn't delete the other one?

@wild-endeavor They call it "MasterName" in redis config but I was trying to avoid using "Master" in our own config...

Reply:
i guess time to support deprecation in config